_posts/2023-03-11-release.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Release 2023.10
+author: Priya
+date: 2023-03-11
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+#### Revamped BigQuery Connector for Better User Experience
+* Now, users can remove and edit a BigQuery service file in the export section. Also, multiple service files can be added.
```

Comment:
Introduced a feature called Smart combine. This allows users to merge new data into an existing exported table. If the same set of key columns is reported, the corresponding values are updated. If a new set of key columns come in, they are now added to the table. Supported on Big Query.

Reply:
added